src/server/components/dashboard/listsItems/helpers.ts

```diff
@@ -22,9 +22,9 @@ export async function redirectIfUnauthorised(req: Request, res: Response, next:
       },
     })) as List;
 
-    const userCanPublishList = userIsListPublisher(req, listData);
+    const authUser = new AuthenticatedUser(req?.user?.userData as User);
```

Comment:
can't you just call `req.user.isListPublisher` directly instead of create a new object?

Reply:
An interesting, I didn't know about that. Guess the object is already created early on in the app.